apps/front/forms.py

```diff
@@ -9,7 +9,7 @@
 from registration.forms import RegistrationForm
 
 
-USERNAME_RE = re.compile(r'^[a-zA-Z-_]+$')
+USERNAME_RE = re.compile(r'^[a-zA-Z0-9-_]+$')
```

Comment:
Das war noch ein Bug in https://github.com/studentenportal/web/commit/b6c0c77320405416707bd67610b47286fc5f1d5e was aber bisher nie deployed wurde - cc @dbrgn